ports/vcpkg-cmake/vcpkg.json

```
@@ -1,5 +1,5 @@
 {
   "name": "vcpkg-cmake",
-  "version-date": "2021-02-28",
-  "port-version": 3
+  "version-date": "2021-06-25",
```

```suggestion
  "version-date": "2021-02-28",
```
This might not be updated.

---

I _could_ change this, but that particular change comes from another PR: #18156 

I built this PR based on that, since without that PR multi-arch builds in vcpkg are completely broken, regardless of the ports themselves. Assuming we still want to merge the other PR - and not just merge this one and close the other, we should fix the other PR and then I can rebase this one again.

I'm OK either way, so please just let me know what is best here.

---

Thanks for your clarification. I remembered I saw these changes before this, but I'm not sure where these changes are. Thanks for your information. So you changes need to reply on the changes in PR #18156, right?

If so, you don't need to update this. Let's wait for PR #18156 merged. Then you can rebase this one.